src/test/java/com/easypost/OrderTest.java

```diff
@@ -169,7 +173,57 @@ public void testNewRate() throws EasyPostException {
      */
     @Test
     public void testLowestRate() throws EasyPostException {
-        vcr.setUpTest("lowest_rate");
+        final String orderResponseJson = "{\n" + "  \"rates\": [\n" + "    {\n" +
```

Comment:
thought: I'm not convinced this is an improvement. Having long concatenated strings like this incredibly messy and difficult to deal with.

Reply:
The mock client can also take in an Object and serialize it to JSON if you want something cleaner.

Reply:
I'd take objects over strings every day of the week. I'd also suggest trimming this down to the bare minimum we need to get the test working (no need to include a bunch of extra hard-coded data when it's not needed)